src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/dataBrokerProfiles.module.scss

```diff
@@ -139,7 +139,7 @@
   font: $text-body-sm;
   margin-bottom: $spacing-md;
 
-  button {
+  .toggleBtn {
```

Comment:
Actually, and you're going to hate me for only realising this now, but... There's actually a `<BillingPeriodToggle>` component used on the landing page that implements radio buttons :facepalm: It looks almost exactly the same as your implementation, so that's good i guess - hope you didn't spend too much time on implementing it again :cry: